pom.xml

```diff
@@ -55,7 +55,7 @@
         <reports.repository.url/>
         <project.scm.id>github</project.scm.id>
         <!-- Must MANUALLY update this if the ddf/support project's version changes -->
-        <ddf.support.version>2.3.6</ddf.support.version>
+        <ddf.support.version>2.3.7-SNAPSHOT</ddf.support.version>
```

Comment:
@shaundmorris Do you usually move to SNAPSHOT or do we just release support and directly move to the next released version?